avocado/core/dispatcher.py

```diff
@@ -121,4 +124,84 @@ def __init__(self):
 class InitDispatcher(EnabledExtensionManager):
 
     def __init__(self):
+        try:
+            help_msg = 'Init extension dispatcher'
+            future_settings.register_option(section='plugins.init',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
+                                            help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+
+        try:
+            help_msg = 'CLI extension dispatcher'
+            future_settings.register_option(section='plugins.cli',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
+                                            help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+
+        try:
+            help_msg = 'CLI command extension dispatcher'
+            future_settings.register_option(section='plugins.cli.cmd',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
+                                            help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+
+        try:
+            help_msg = 'Job Pre/Post extension dispatcher'
+            future_settings.register_option(section='plugins.job.prepost',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
+                                            help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+        try:
+            help_msg = 'Result extension dispatcher'
+            future_settings.register_option(section='plugins.result',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
+                                            help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+
+        try:
+            help_msg = 'Result Events extension dispatcher'
+            future_settings.register_option(
+                section='plugins.result_events',
+                key='order',
+                key_type=list,
+                default=[],
+                help_msg=help_msg)
+        except DuplicatedNamespace:
+            pass
+
+        try:
+            help_msg = 'Varianter extension dispatcher'
+            future_settings.register_option(section='plugins.varianter',
+                                            key='order',
+                                            key_type=list,
+                                            default=[],
```

Comment:
Instead of multiple try, could you have all the register_option inside one try.. except?

Reply:
Yes, it is possible and makes sense, sorry about that, it was copy/paste from each specific class I tried the first time.